src/backend/aspen/workflows/import_lineages/load_lineages.py

```diff
@@ -0,0 +1,168 @@
+from typing import Any, List
+
+import click
+import requests
+import sqlalchemy as sa
+
+from aspen.config.config import Config
+from aspen.database.connection import (
+    get_db_uri,
+    init_db,
+    session_scope,
+    SqlAlchemyInterface,
+)
+from aspen.database.models import Pathogen, PathogenLineage
+from aspen.util.pathogen_configs import get_lineage_urls
+from aspen.workflows.shared_utils.database import (
+    create_temp_table,
+    drop_temp_table,
+    mv_table_contents,
+)
+
+
+def download_lineages(
+    url: str,
+    format: str,
+    print_response: bool,
+) -> Any[str, List[str]]:
```

Comment:
What does this mean? I've never seen this construct before with `Any`, I didn't think it could take ... args? types? type-args? I have no idea what to generically call the X in `List[X]`, but that thing. I thought `Any` stood alone? I could totally be wrong and learning something new here though!